src/index.html

```diff
@@ -11,8 +11,49 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
```

Comment:
You should include a CSS file in your HTML, not a SCSS file. SCSS files need to be compiled to CSS before being used in an HTML file.